mammoth/utils/optimizers.py

```diff
@@ -350,6 +346,18 @@ def report_steps(self):
             result.append(f'Optimizer "{name}" has been stepped {count} times and has LR {lr}')
         return result
 
+    def count_parameters(self, log=print):
```

Comment:
this is inviting a deluge of random useless prints. use the logger properly and put the info at a very fine level, like debug...

Reply:
The actual calls look like this
```enc, dec = model.count_parameters(log=logger.debug)```
I don't know who needed to print this instead, and why. I'll nuke it.